### ISSUE TITLE
DTP feature size in your code is different from that said in paper.

### ISSUE THREAD

**Issue by Senwang98** [olly-styles/Multiple-Object-Forecasting#10]
There is a question about your WACV paper, you said the output size is 2048 after FC-2, but your code dtp_feature's is 2048 already and the output of FC-2 is 512/2. I don't know if i understood wrong.

Code:
`self.dtp_encoder = nn.Linear(2048, int(self.num_hidden / 2))`

Paper:
![屏幕截图](https://user-images.githubusercontent.com/23122139/88507973-37a78100-d010-11ea-8fd1-c2af5344f92a.png)

**Comment by olly-styles** [comment]
Good spot,  I agree this is misleading. 2048 is the size of the feature vector obtained from the optical flow, which is then reduced to 256 by FC2. Likewise, the 2048 size feature vector from the bounding box encoder is reduced to 256 by 256. Therefore, the size of φc is 512. In Figure 4, the φb and φf labels should be before the fully connected layers, not after.
